package.json

```diff
@@ -20,24 +20,38 @@
     "url": "https://github.com/metarhia/globalstorage"
   },
   "main": "gs.js",
+  "browser": {
+    "gs.js": "dist/gs.js",
+    "lib/fs.cursor.js": false,
+    "lib/fs.provider.js": false
+  },
   "scripts": {
     "test": "npm run lint && node test/all",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "build": "babel gs.js -d dist && babel lib -d dist/lib --ignore lib/fs.provider.js,lib/fs.cursor.js",
+    "prepublish": "npm run -s build"
   },
   "engines": {
     "node": ">=6.0.0"
   },
   "dependencies": {
     "mdsf": "^1.0.1",
     "metarhia-common": "^0.0.32",
-    "metasync": "^0.3.29",
+    "metasync": "^0.3.29"
+  },
+  "optionalDependencies": {
     "mkdirp": "^0.5.1"
   },
   "devDependencies": {
     "eslint": "^5.5.0",
     "eslint-config-metarhia": "^4.0.0",
     "eslint-plugin-import": "^2.14.0",
     "metaschema": "^0.0.21",
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/runtime": "^7.0.0",
```

Comment:
Either this must be a `dependency`, or `plugin-transform-runtime` must not be used at all.